check-disk/lib/check-disk.go

```diff
@@ -46,17 +46,13 @@ func checkStatus(current checkers.Status, threshold string, units float64, disk
 		}
 
 		freePct := float64(100) - disk.UsedPercent
-
 		inodesFreePct := float64(100) - disk.InodesUsedPercent
-		if float64(disk.InodesTotal) == float64(0) {
-			inodesFreePct = float64(0)
-		}
 
-		if chkInode && v > inodesFreePct {
+		if chkInode && (disk.InodesTotal != 0 && v > inodesFreePct) {
 			current = status
```

Comment:
I did not understand what disk.InodesTotal! = 0 means for just a moment.  So How about writing comment, for example `If InodesTotal == 0, the system don't use inode`

Reply:
I added a note 94e20dc :+1: